storybook/stories/Examples/Pie/PieWithNeedle.stories.tsx

```diff
@@ -68,3 +68,40 @@ export const PieWithNeedle = {
   },
   args: {},
 };
+
+export const PieWithPatterns = {
```

Comment:
For consistency I would expect to find this story also in the API section, i.e. in file 

https://github.com/recharts/recharts/blob/master/storybook/stories/API/polar/Pie.stories.tsx

Could you please move it there?